src/components/TextOrInput/styled.js

```diff
@@ -1,7 +1,28 @@
 import styled from 'styled-components';
+import Icon from '../Icon';
+import Input from '../Input';
 
 export const Wrap = styled.div`
   position: relative;
+  display: flex;
+  align-items: center;
+
+  ${({ isInput, canEdit }) =>
+    !isInput && canEdit ? 'cursor: pointer' : !canEdit && 'cursor: default'};
```

Comment:
can you simplify it? cause we only need to make a `pointer` for one case
my proposal is – `${({ isInput, canEdit }) => !isInput && canEdit && 'cursor: pointer'};`

Reply:
fixed